awx/main/access.py

```diff
@@ -2550,10 +2508,6 @@ class LabelAccess(BaseAccess):
     def filtered_queryset(self):
         return self.model.objects.all()
 
-    @check_superuser
-    def can_read(self, obj):
-        return self.user in obj.organization.read_role
```

Comment:
So this is the only real issue here. We list all labels in `/api/v2/labels/`, but you can't visit `/api/v2/labels/N/` unless you're a member of its organization. Also, because of this, you can't attach labels if you are not in its organization.

I cannot imagine any reason for those combinations of behavior. This does very little to enforce parity between the JT organization and the label organization.

After this change, we will have the problem of duplicate labels of the same name in multiple orgs. But we have that either way. I try, and I can't think of any really good reason to keep this around.